feat: add avif to image extensions (#3591)
plentymarkets/plugin-ceres: meta/documents/changelog_en.md

# Release Notes for plentyShop LTS

### Fixed

> - Images in AVIF format could not be selected in the Background Image widget. This has beeen adressed, the background image widget shows a fallback input field.

Reviewer: ```suggestion
- In the Background Image widget, it wasn't possible to select AVIF images. This has been fixed by providing a fallback input field.
```
